.github/workflows/composer-validate.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
```

Comment:
Maybe unnecessary. Sounds that this file was generated from a template to create an Action.

```suggestion
```